hokusai/production.yml

```diff
@@ -0,0 +1,102 @@
+---
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: doppler-web
+  namespace: default
+spec:
+  replicas: 3
+  strategy:
+    rollingUpdate:
+      maxSurge: 0
+      maxUnavailable: 1
+    type: RollingUpdate
+  template:
+    metadata:
+      labels:
+        app: doppler
+        component: web
+        layer: application
+      name: doppler-web
+    spec:
+      containers:
+        - name: doppler-web
+          env:
+            - name: RAILS_ENV
+              value: production
+            - name: RAILS_SERVE_STATIC_FILES
+              value: 'true'
+            - name: RAILS_LOG_TO_STDOUT
+              value: 'true'
```

Comment:
I recommend these go into the associated config map (`doppler-environment`) so they're automatically applied to all other commands (e.g., `rails console`).

Reply:
https://github.com/artsy/doppler/pull/160